src/UI/Cells/Edit/QualityCellEditor.js

```diff
@@ -23,7 +23,7 @@ module.exports = Backgrid.CellEditor.extend({
         promise.done(function() {
             var templateName = self.template;
             self.schema = profileSchemaCollection.first();
-            debugger;
+            //debugger;
```

Comment:
Can you remove the debugger statements while you are at it?

Reply:
Well, I found a bunch of them commented out so I commented those too instead of removing them but I can remove all of them if you want.